reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml

```diff
@@ -25,12 +30,6 @@ spec:
           value: "/reana"
         - name: ORGANIZATIONS
           value: {{ORGANIZATIONS}}
-        - name: WDB_SOCKET_SERVER
```

Comment:
good that we remove this from here, but shouldn't it go to the corresponding section for environment variables for RWC in the reana cluster dev file? also applies for all the components containing `WDB_*` env variables.

Reply:
Oh, my bad. I accidentally deleted WDB_* and FLASK_DEBUG envvars.
I will fix that either by including the envvars hardcoded in the K8S manifest templates or make them configurable through `reana-cluster.yaml` / `reana-cluster-dev.yaml`